### PR TITLE
chore(functions/log_helloworld): mention var_dump

### PR DESCRIPTION
**Pull request by ace-n**
Ran into this while debugging; figured it was worth documenting.

### CODE REVIEW

File: functions/log_helloworld/index.php [hunk 1]
```diff
@@ -34,8 +34,9 @@ function helloLogging(ServerRequestInterface $request): string
       'severity' => 'error'
     ]) . PHP_EOL);
 
-    // This doesn't log anything
+    // These doesn't log anything
```
Comment (bshaffer) [inline functions/log_helloworld/index.php:37]: should be "these don't log anything"

Comment (ace-n) [reply functions/log_helloworld/index.php]: Done.

**Comment by bshaffer** [comment]
This has to do with @grayside and my changes to google/cloud-tools , but kokoro has the following error:

```
InvalidArgumentException: Missing required property: entryPoint in /tmpfs/src/github/php-docs-samples/testing/vendor/google/cloud-tools/src/TestUtils/GcloudWrapper/CloudFunction.php:83
Stack trace:
#0 /tmpfs/src/github/php-docs-samples/testing/vendor/google/cloud-tools/src/TestUtils/GcloudWrapper/CloudFunction.php(111): Google\Cloud\TestUtils\GcloudWrapper\CloudFunction->__construct('php-docs-sample...', '', '', '', '')
#1 /tmpfs/src/github/php-docs-samples/testing/vendor/google/cloud-tools/src/TestUtils/CloudFunctionLocalTestTrait.php(58): Google\Cloud\TestUtils\GcloudWrapper\CloudFunction::fromArray(Array)
#2 /tmpfs/src/github/php-docs-samples/testing/vendor/phpunit/phpunit/src/Framework/TestSuite.php(703): Google\Cloud\Samples\Functions\HelloLogging\Test\SystemTest::startFunction()
#3 /tmpfs/src/github/php-docs-samples/testing/vendor/phpunit/phpunit/src/Framework/TestSuite.php(746): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#4 /tmpfs/src/github/php-docs-samples/testing/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(652): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#5 /tmpfs/src/github/php-docs-samples/testing/vendor/phpunit/phpunit/src/TextUI/Command.php(206): PHPUnit\TextUI\TestRunner->doRun(Object(PHPUnit\Framework\TestSuite), Array, true)
#6 /tmpfs/src/github/php-docs-samples/testing/vendor/phpunit/phpunit/src/TextUI/Command.php(162): PHPUnit\TextUI\Command->run(Array, true)
#7 /tmpfs/src/github/php-docs-samples/testing/vendor/phpunit/phpunit/phpunit(61): PHPUnit\TextUI\Command::main()
```

**Comment by ace-n** [comment]
@bshaffer I suspect that's an issue with the tests (that should likely be remedied in a separate PR).

**Review by grayside — CHANGES_REQUESTED** [review]
could not reproduce this problem: I deploy a function, called it, and as expected the var_dump() output was returned in the HTTP response.

**Comment by ace-n** [comment]
Ah - I didn't think to check there, my bad!

(I clarified the note a bit more, so @grayside PTAL.)

**Review by grayside — APPROVED** [review]
Filing a follow-up issue, as I think this is expanding the scope of the sample in a way that is both helpful but invites further details.

**Comment by grayside** [comment]
#1219

**Review by bshaffer — APPROVED** [review]

**Pull request merged**